libcudacxx/test/libcudacxx/std/algorithms/alg.modifying/alg.copy/copy_if.pass.cpp

```diff
@@ -118,10 +118,6 @@ TEST_CONSTEXPR_CXX14 __host__ __device__ bool test()
 int main(int, char**)
 {
   test();
-
-#if TEST_STD_VER >= 2014 && !defined(TEST_COMPILER_MSVC_2017)
```

Comment:
Not happy about the Standard checks being removed, we want them to be a different PR in case we need to roll back